pdf/pdfcanvas.cpp

```diff
@@ -348,12 +357,14 @@ QPointF PDFCanvas::fromPageToItem(int index, const QPointF &point) const
                    point.y() * page.rect.height() + page.rect.y());
 }
 
-void PDFCanvas::pageFinished(int id, QSGTexture *texture)
+void PDFCanvas::pageFinished(int id, int resolution, QRectF subpart, QSGTexture *texture)
```

Comment:
You're right. I kept ```pageRenderWidth``` to emphase that this is the total page render width and not the texture render width.